shared/src/commonMain/kotlin/com/bunbeauty/shared/data/mapper/user_address/UserAddressMapper.kt

```diff
@@ -40,9 +38,9 @@ class UserAddressMapper(
     fun toUserAddressEntity(addressServer: AddressServer): UserAddressEntity {
         return UserAddressEntity(
             uuid = addressServer.uuid,
-            streetUuid = addressServer.street.uuid,
-            streetName = addressServer.street.name,
-            cityUuid = addressServer.street.cityUuid,
+            streetUuid = "",
```

Comment:
is it correct?

Reply:
It's correct. We don't need this field anymore. 
✅ fixed - remove it from Address table